unbalanced_dataset/base.py

```diff
@@ -181,3 +182,25 @@ def fit_sample(self, X, y):
         """
 
         return self.fit(X, y).sample(X, y)
+
+
+    def transform(self, X):
```

Comment:
Let me explain this. When you call the transform method in pipeline it supposed that the pipiline is already fitted. In the fit method the _pre_transform method it is called which performs the fit_sample. So, no need for sample in transform. Am I right?